bower.js

```diff
@@ -1,12 +1,20 @@
 var path = require("path");
 var exec = require("./exec").exec;
 
-module.exports = function(pro, args, callback) {
-  var bowerArgs = args.remainder;
-  var executable = "bower";
-  if (process.platform == "win32") {
-  	executable += ".cmd";
-  }
-  exec(path.join(__dirname, "node_modules", ".bin", executable), false,
-       bowerArgs, process.env, callback);
-};
+module.exports = (function() {
```

Comment:
So it's clear, I did this this way so that the external API wouldn't change, but I wanted to add `launchBower` so that I didn't have to construct an artificial `args.remainder` object.